platform.txt

```diff
@@ -129,3 +129,26 @@ tools.openocd.erase.pattern=
 tools.openocd.bootloader.params.verbose=-d2
 tools.openocd.bootloader.params.quiet=-d0
 tools.openocd.bootloader.pattern="{path}/{cmd}" {bootloader.verbose} -f interface/{program.protocol}.cfg -c "{program.setup_command}" -f target/{upload.target}.cfg -c "init; halt; nrf51 mass_erase; program {{{runtime.platform.path}/cores/nRF5/SDK/components/softdevice/{softdevice}/hex/{softdevice}_{upload.target}_{softdeviceversion}_softdevice.hex}} verify reset; shutdown;"
+
+# blackmagic probe upload
+tools.bmp_upload.cmd=arm-none-eabi-gdb
```

Comment:
A more descriptive name than `bmp_upload` would be nice, maybe `blackmagicprobe`?

Thoughts?

Reply:
@sandeepmistry sounds good